templates/default/ckan_properties.ini.erb

```diff
@@ -126,6 +126,9 @@ ckan.harvest.mq.type = redis
 ckan.harvest.mq.hostname = <%= node['datashades']['redis']['hostname'] %>
 ckan.harvest.mq.port = <%= node['datashades']['redis']['port'] %>
 
+## QA
+qa.resource_format_openness_scores_json = <%= @src_dir %>/ckanext-data-qld/ckanext/data_qld/resource_format_openness_scores.json
```

Comment:
hmm, do we have 2 different ckan ini files for publications/data. having extra things should not affect the other right?

Reply:
We have only one file template, but unrecognised properties are simply ignored. We've had unused properties sitting in the file since day one.